clang/cmake/caches/Fuchsia-stage2.cmake

```diff
@@ -28,6 +28,7 @@ set(LLVM_STATIC_LINK_CXX_STDLIB ON CACHE BOOL "")
 set(LLVM_USE_RELATIVE_PATHS_IN_FILES ON CACHE BOOL "")
 set(LLDB_ENABLE_CURSES OFF CACHE BOOL "")
 set(LLDB_ENABLE_LIBEDIT OFF CACHE BOOL "")
+set(LIBCXX_TEST_PARAMS "std=c++23" CACHE STRING "")
```

Comment:
This needs to be set separately for all targets, i.e. `RUNTIMES_${target}_LIBCXX_TEST_PARAMS`, `LIBCXX_TEST_PARAMS` should be only set for Darwin.

Reply:
You are correct. I have set `set(RUNTIMES_${target}_LIBCXX_TEST_PARAMS "std=c++23" CACHE STRING "")` in windows runtime target in patch 24f78f125b4f7a9ea4f3488d0acc396b510c86c8 . Since these are windows only tests.